test/e2e/tests/FormTest.js

```diff
@@ -87,6 +101,9 @@ export default {
       .publishFormOptions()
       .getUrlStandaloneForm(({ url }) =>{
 
+        // Saving FORM ID
+        testData.FORM_ID = url.match(/(\S{24}).html$/)[1]
```

Comment:
so i think you're grabbing the objectid here. a better RegExp would be:

`/([a-f\d]{24})\.html$/i`. Test it to be sure probably